### PR TITLE
merge core changes from stt

### PR DESCRIPTION
**Pull request by petrjasek**

### CODE REVIEW

File: data/ui_config.json [hunk 1]
```diff
@@ -0,0 +1,77 @@
+[
```
Comment (MarkLark86) [inline data/ui_config.json:1]: Should we include this config in core?

File: newsroom/commands/initialize_data.py [hunk 2]
```diff
@@ -22,7 +24,10 @@ def run(self, entity_name=None, force=False, init_index_only=False):
         # create indexes in mongo
         app.init_indexes()
         # put mapping to elastic
-        app.data.init_elastic(app)
+        try:
+            app.data.init_elastic(app)
+        except elasticsearch.exceptions.TransportError as err:
+            logger.error("Error when initializing elastic %s", err)
```
Comment (MarkLark86) [inline newsroom/commands/initialize_data.py:30]: Any idea about this error:
```bash
Error when initializing elastic TransportError(500, 'mapper_exception', "Can't update attribute for type [_doc.extra.enabled] in index mapping")
```

File: newsroom/template_filters.py [hunk 3]
```diff
@@ -38,7 +38,7 @@ def datetime_long(datetime):
 
 
 def date_header(datetime):
-    return format_datetime(parse_date(datetime if datetime else utcnow()), 'EEEE, MMMM d, yyyy')
+    return format_datetime(parse_date(datetime if datetime else utcnow()), 'EEEE, dd.MM.yyyy')
```
Comment (MarkLark86) [inline newsroom/template_filters.py:41]: This format should really be in a config somewhere

File: package-lock.json [hunk 4]
```diff
@@ -328,14 +328,113 @@
       }
```
Comment (MarkLark86) [inline package-lock.json:328]: Do we need a package-lock in development branches

File: webpack.config.js [hunk 5]
```diff
@@ -3,32 +3,31 @@
...
+        cards_js: path.resolve(__dirname, './assets/cards/index.js'),
+        user_profile_js: path.resolve(__dirname, './assets/user-profile/index.js'),
+        newsroom_css: path.resolve(__dirname, './assets/style.js'),
+        wire_js: path.resolve(__dirname, './assets/wire/index.js'),
+        home_js: path.resolve(__dirname, './assets/home/index.js'),
+        agenda_js: path.resolve(__dirname, './assets/agenda/index.js'),
+        notifications_js: path.resolve(__dirname, './assets/notifications/index.js'),
+        company_reports_js: path.resolve(__dirname, './assets/company-reports/index.js'),
+        print_reports_js: path.resolve(__dirname, './assets/company-reports/components/index.js'),
+        am_news_js: path.resolve(__dirname, './assets/am-news/index.js'),
```
Comment (MarkLark86) [inline webpack.config.js:24]: We should probably remove AAP specific sections at some point

File: newsroom/commands/initialize_data.py (hunk 2 above)
Comment (petrjasek) [reply newsroom/commands/initialize_data.py:30]: yep that's due to reusing previous index, needs reindexing, but first I need updated instance

File: data/ui_config.json (hunk 1 above)
Comment (petrjasek) [reply data/ui_config.json]: seems like it needs some default value, will update it

File: newsroom/template_filters.py (hunk 3 above)
Comment (MarkLark86) [reply newsroom/template_filters.py:41]: What about this one? A simple config with default in `newsroom/web/default_settings.py` should suffice

**Pull request merged**